default/AI/AIstate.py

```diff
@@ -307,77 +258,74 @@ def assess_enemy_supply(self):
                             if sum_stats[0] > 0:
                                 e_f_dict.setdefault(sum_stats, [0])[0] += count
-                    partialVisTurn = universe.getVisibilityTurnsMap(fleetID, empireID).get(fo.visibility.partial, -9999)
-                    if partialVisTurn >= current_turn - 1:  # only interested in immediately recent data
+                    partial_vis_turn = universe.getVisibilityTurnsMap(fleet_id, empire_id).get(fo.visibility.partial, -9999)
+                    if partial_vis_turn >= current_turn - 1:  # only interested in immediately recent data
                         if not dead_fleet:
-                            sawEnemiesAtSystem[fleet.systemID] = True
-                            enemyFleetIDs.append(fleetID)
-                            enemiesBySystem.setdefault(thisSysID, []).append(fleetID)
+                            saw_enemies_at_system[fleet.systemID] = True
+                            enemy_fleet_ids.append(fleet_id)
+                            enemies_by_system.setdefault(this_system_id, []).append(fleet_id)
                             if not fleet.ownedBy(-1):
-                                self.misc.setdefault('enemies_sighted', {}).setdefault(current_turn, []).append(fleetID)
-                                rating = self.rate_fleet(fleetID, self.fleet_sum_tups_to_estat_dicts([(1, self.empire_standard_fighter)]))
+                                self.misc.setdefault('enemies_sighted', {}).setdefault(current_turn, []).append(fleet_id)
+                                rating = self.rate_fleet(fleet_id, self.fleet_sum_tups_to_estat_dicts([(1, self.empire_standard_fighter)]))
                                 if rating.get('overall', 0) > 0.25 * my_milship_rating:
-                                    self.misc.setdefault('dangerous_enemies_sighted', {}).setdefault(current_turn, []).append(fleetID)
+                                    self.misc.setdefault('dangerous_enemies_sighted', {}).setdefault(current_turn, []).append(fleet_id)
         e_f_dict = [cur_e_fighters, old_e_fighters][len(cur_e_fighters) == 1]
         std_fighter = sorted([(v, k) for k, v in e_f_dict.items()])[-1][1]
         self.empire_standard_enemy = std_fighter
         self.empire_standard_enemy_rating = std_fighter[0] * std_fighter[-1]  # using a very simple rating method here
 
-        #assess fleet and planet threats & my local fleets
-        for sys_id in sysIDList:
+        # assess fleet and planet threats & my local fleets
+        for sys_id in system_id_list:
             sys_status = self.systemStatus.setdefault(sys_id, {})
             system = universe.getSystem(sys_id)
-            #update fleets
-            sys_status['myfleets'] = myFleetsBySystem.get(sys_id, [])
-            sys_status['myFleetsAccessible'] = fleetSpotPosition.get(sys_id, [])
-            localEnemyFleetIDs = enemiesBySystem.get(sys_id, [])
-            sys_status['localEnemyFleetIDs'] = localEnemyFleetIDs
```

Comment:
It is my mistake. I missed this change in diff.  This should not be renamed at this moment.

As result we have issue in fleet orders.
This issue happens not often. It affects exploration and invasion fleets after turn 50. It does not repoduced for all empires.

ps.  This comment in code is correct, `can_issue_order` executed twice.
```pyhton
if not self.can_issue_order():  # appears to be redundant with check in IAFleetMission?
```